### PR TITLE
enableMenu (optional)

### PR DESCRIPTION
**Pull request by ResuIl**
## Description

You can set whether or not a special radial appears for any situation.

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -1,51 +1,54 @@
 local config = require 'config.client'
 
 local function convert(tbl)
-    if tbl.items then
-        local items = {}
-        for _, v in pairs(tbl.items) do
-            items[#items + 1] = convert(v)
+    local _, e = pcall(tbl.enableMenu)
+    if type(e) == "string" or e then
```
Comment (BerkieBb) [inline client/main.lua:5]: Why this?

**Comment by TonybynMp4** [comment]
What's the point of this even?

File: client/main.lua (hunk 1 above)
Comment (ResuIl) [reply client/main.lua:5]: enableMenu must return bool

**Comment by ResuIl** [comment]
> What's the point of this even?

in some situations it need

File: client/main.lua (hunk 1 above)
Comment (ResuIl) [reply client/main.lua:5]: This solves the problem that radialmenu will not work if this code is not on this line and enableMenu is not configured

**Comment by mafewtm** [comment]
We won't be adding this at this time.

**Pull request closed without merging**